keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -612,24 +605,27 @@ message ProvisionerGetInfoResponse {
 This call is made to create the bucket in the backend. If a bucket that matches both name and parameters already exists, then OK (success) must be returned. If a bucket by same name, but different parameters is provided, then the appropriate error code `ALREADY_EXISTS` must be returned by the provisioner. The call to *ProvisonerCreateBucket* MUST be idempotent.
 
 ```
-message ProvisionerCreateBucketRequest {    
-    // This field is REQUIRED
-    string bucket_name = 1;
-
-    map<string,string> bucket_context = 2;
+message ProvisionerCreateBucketRequest {
+    map<string,string> bucket_context = 1;
```

Comment:
How does the sidecar know which key in `bucket_context` is for the bucket name? Or is this field opaque to the sidecar?

Reply:
Is the plan to move `bucket_name` to `bucket_context`?  Why is that?

Reply:
Yes.  The driver has to handle 3 cases:

Brownfield: A bucketname will be passed to the driver
Greenfield with prefix: No bucketname, but a prefix is passed to the driver.  driver creates bucket name with prefix
Greendfield no prefix: No bucketname nor prefix, driver creates bucketname

Since neither bucketname nor prefix can be guaranteed to exist in any request, we moved it and that data will be in bucket_context.

Reply:
@jeffvance We will have to document what fields we know will be in the bucket_context.  Specifically, we will have to detail the name parameters and what to do in each case.

Reply:
For Brownfield, a bucket already exists, so CreateBucket should not be called.  It should just be a binding operation.

Reply:
Update to this in the next update.  There will be an id field in the ProvisionerCreateBucketRequest